packages/e2e-tests/plugins/allowed-patterns.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Plugin Name: Gutenberg Test Allowed Patterns
+ * Plugin URI: https://github.com/WordPress/gutenberg
+ * Author: Gutenberg Team
+ *
+ * @package gutenberg-test-allowed-patterns
+ */
+
+register_block_pattern(
+	'test-allowed-patterns/lone-heading',
+	array(
+		'title'   => 'Test: Single heading',
+		'scope'   => array(
+			'inserter' => true,
+		),
+		'content' => '<!-- wp:heading --><h2>Hello!</h2><!-- /wp:heading -->',
+	)
+);
+
+register_block_pattern(
+	'test-allowed-patterns/lone-paragraph',
+	array(
+		'title'   => 'Test: Single paragraph',
+		'scope'   => array(
```

Comment:
experimental `scope` has been removed from patterns (https://github.com/WordPress/gutenberg/pull/30471)

Reply:
Thanks! Removed it and rebased